R/visr_plot.R

```diff
@@ -258,10 +259,24 @@ visr.survfit <- function(
                                 limits = c(min(y_ticks), max(y_ticks))) +
     ggplot2::theme(legend.position = legend_position) +
     NULL
-
+  
   class(gg) <- append(class(gg), "ggsurvfit")
   
-  return(gg)
+  if (theme == "visR") {
```

Comment:
after cran:

look for a globally defined theme => if not present go for visR theme